commit fb4f4c44ed487faeb534eaf1895a3cb9b07174d6
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-25T10:29:37-07:00 (2 months ago)
    
    TRITON-1828 drop firmware-tools and "headnode-joyent" flavour headnode build (change jr metadata for sdc-headnode to have the build component name be 'headnode')
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/tools/jr-manifest.json b/tools/jr-manifest.json
index 67fdd26..6ee8a81 100644
--- a/tools/jr-manifest.json
+++ b/tools/jr-manifest.json
@@ -646,8 +646,7 @@
             "name": "sdc-headnode",
             "labels": {
                 "release": true,
-                "mg": "headnode-joyent",
-                "buildisprivate": true
+                "mg": "headnode"
             }
         },
         {
